src/store/constants.js

```diff
@@ -12,6 +14,27 @@ export const JOB_TYPE_MAP = {
   4: 'Freelance',
 };
 
+export const PERIODS = [
+  {
+    type: 'daily',
+    text: 'Bugün',
+    slug: 'bugun',
+    isDefault: false,
```

Comment:
You can remove `isDefault: false`. It will only match `isDefault: true` one so it doesn't have to be set to false explicitly.